tests/integration_tests/deployment_template.toml

```diff
@@ -0,0 +1,22 @@
+ip = "172.17.0.1"
+log_level = "INFO"
+celery_log_level ="INFO"
+monetdb_image = "madgik/mipenginedb_with_data:dev1.2"
+
+[[nodes]]
+id = "globalnode"
+role="globalnode"
```

Comment:
GLOBALNODE

Reply:
Does it make sense that this is working with lower case?